exonum/src/runtime/mod.rs

```diff
@@ -262,6 +262,9 @@ pub trait Runtime: Send + Debug + 'static {
     ///
     /// This method is called during an API server restart. Use this method if you do not plan to
     /// use your own API processing mechanism.
+    /// 
+    /// Warning! It is a temporary method which not to break the existing `RustRuntime` code,
```

Comment:
"It is a temporary method [that exists/existing/present] to not break the existing `RustRuntime` code,"? Maria might suggest a better option. Though it is internal thing, and we as intended audience understand it ^_^